src/components/10-atoms/input-file/ui.test.js

```diff
@@ -0,0 +1,89 @@
+import { Selector, ClientFunction } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL || 'http://localhost:9999';
+
+fixture('Input File - basic functionality').page(
+  `${host}/iframe.html?id=atoms-input-file--input-file-default`
+);
+
+const INPUT_FILE_TAG = 'axa-input-file';
+const ICON_TAG = 'axa-icon';
+const INPUT_FILE_CLASS = '.a-input-file';
+const INPUT_FILE_INPUT_CLASS = '.a-input-file__input';
+
+test('should render input-file', async t => {
+  const $axaElem = await Selector(INPUT_FILE_TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaElemEl = await $axaElem.find(INPUT_FILE_CLASS);
+  await t.expect($axaElemEl.exists).ok();
+});
+
+fixture('Input File - set property icon').page(
+  `${host}/iframe.html?id=atoms-input-file--input-file-icon`
+);
+
+test('should set input file property icon', async t => {
+  const $axaElem = await Selector(INPUT_FILE_TAG);
+  await t.expect($axaElem.exists).ok();
+  const $axaIcon = await $axaElem.find(ICON_TAG);
+  await t.expect($axaIcon.exists).ok();
+});
+
+fixture('Input File - set property disabled').page(
+  `${host}/iframe.html?id=atoms-input-file--input-file-default`
+);
+
+test('should set input file property disabled', async t => {
+  const setDisabled = ClientFunction(() => {
+    document.querySelector('axa-input-file').disabled = true;
+  });
+  await setDisabled();
+  const $axaElm = await Selector('axa-input-file');
+  await t.expect(await $axaElm.hasAttribute('disabled')).ok();
```

Comment:
I guess this test can be removed. It doesn't really add value imho. This will always be true and if not, the element has a problem which will be detected by other tests.

Reply:
True, @NoahLiechti sorry here you need to get the value of the input inside axa-input-file. this is valid for all of them